lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -1,4 +1,8 @@
 <?php
+
+use DoSomething\Gateway\Contracts\TransactionBridgeContract;
```

Comment:
@katiecrane did we decide we didn't need this here and only needed 
```
use DoSomething\Gateway\Common\RestApiClient;
use DoSomething\Gateway\ForwardsTransactionIds;
```
per our [slack convo](https://dosomething.slack.com/archives/D0F16UE2D/p1475871108000074)?